scriptoocloud/src/main/resources/error.properties

```diff
@@ -0,0 +1,5 @@
+400 = 400. Bad Request
+401 = 401. Unauthorized
+403 = 403. Access Denied
+404 = 404. Page Not Found
+500 = 500. Internal Server Error
```

Comment:
Let's make these a little bit more user friendly, something like "400=The server was unable to understand your browser's request".